server/src/main/java/io/unitycatalog/server/UnityCatalogServer.java

```diff
@@ -92,8 +93,27 @@ private void addServices(ServerBuilder sb) {
 
   public static void main(String[] args) {
     int port = 8080;
-    if (args.length > 0) {
-      port = Integer.parseInt(args[0]);
+    Options options = new Options();
+    options.addOption(
+            Option.builder("p")
+                    .longOpt("port")
+                    .hasArg()
+                    .desc("Port number to run the server on. Default is 8080.")
+                    .type(Integer.class)
+                    .build());
+    CommandLineParser parser = new DefaultParser();
+    try {
+      CommandLine cmd = parser.parse(options, args);
+      if (cmd.hasOption("p")) {
+        port = cmd.getParsedOptionValue("p");
+      }
+    } catch (ParseException e) {
+      System.out.println();
+      System.out.println("Parsing Failed. Reason: " + e.getMessage());
+      System.out.println();
+      HelpFormatter formatter = new HelpFormatter();
+      formatter.printHelp("bin/start-uc-server", options);
```

Comment:
Can we please update the docs with these details? i.e. how to start the server on a specific port?

Reply:
PTAL